### ISSUE TITLE
API level 25 on Android might be required 

### ISSUE THREAD

**Issue by LarkAscending** [flutter/friendlychat-steps#39]
Steps to Reproduce
1. Install Android API level 26 (only).
1. Load full_steps/step_0 sample code.
1. Click Run in IntelliJ

Get error message.

Try installing 25 and agree to the SLA and see if the error disappears.
/cc @efortuna @emshack

**Comment by LarkAscending** [comment]
@efortuna was this the error message?
```
E/flutter ( 8442): [ERROR:flutter/shell/gpu/gpu_surface_gl.cc(189)] WARNING: Could not create offscreen surface. This device or emulator does not support color correct rendering. Fallbacks are in effect. Colors on this device will differ from those displayed on most other devices. This warning will only be logged once.
```
I tried uninstalling 25 and installing 26, but they seem to be linked now. 25 shows as "Partially installed" and 26 shows "Update available" but can't be selected in the list.  On the SDK Tools tab, the "latest" Android SDK Build-Tools are 25.0.3+26.0.2 (when uninstalling you get this info). It's possible the Android Studio team is trying to address the dependency you found...

My SDK Platforms are in a weird limbo state. I could find no issue logged against sdkmanager describing this. Currently unable to complete the repro steps...can you repro?

**Comment by LarkAscending** [comment]
Today it looks like API level 27 became available in SDK Manager. After installing its status is "Partially Installed" which is weird. 

Using API 27 Friendlychat seems to run/work normally and the GPU surface error (above) does not seem to affect that. It's just a warning... Leaving open in case we repro and capture the real error message, or find that Friendlychat doesn't work normally on the latest Android SDK Tools (27).

**Comment by LarkAscending** [comment]
@efortuna I used the CLI tool sdkmanager to remove API level 25. FYI 26 and 27 were installed.

What I found is that although it IS possible to remove the API 25 build tools from the command line, running the app from IntelliJ somehow adds them back again. So it seems there is a dependency and you found it before the AS folks had addressed it. Today it seems to be addressed.

LMK if you agree and we'll close this issue. We can re-open if you repro the original error (which I never saw during this process).
 I think it's addressed.

LMK if you agree and we'll close this issue. We can re-open if you repro the original error (which I never saw during this process).
